.github/workflows/linux_builds.yml

```diff
@@ -4,7 +4,7 @@ on: [push, pull_request]
 # Global Settings
 env:
   GODOT_BASE_BRANCH: master
-  SCONSFLAGS: platform=linuxbsd verbose=yes warnings=extra werror=yes debug_symbols=no --jobs=2 module_text_server_fb_enabled=yes
```

Comment:
I'd keep this here to be consistent with all other platforms. You can still override with `debug_symbols=yes` for the sanitizer build.